runtime/doc/options.txt

```diff
@@ -1307,6 +1307,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 	The value of this option is stored with the tab page, so that each tab
 	page can have a different value.
 
+	When 'cmdheight' is zero, it disables echo area and all outputs needs
```

Comment:
```suggestion
	When 'cmdheight' is 0, it disables echo area and all outputs needs
```

I think we should use `0` here instead of `zero` for consistency with other places.